src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -70,13 +72,17 @@ export const ProjectListItemCard: FC<ProjectListItemCardProps> = (props) => {
     <Card className={clsx(classes.root, props.className)}>
       <CardActionAreaLink to={`/projects/${props.id}`} className={classes.card}>
         <div className={classes.media}>
-          <Picture
-            source={pic}
-            fit="cover"
-            width={300}
-            height={200}
-            onError={nextPic}
-          />
+          {props.loading ? (
+            <Skeleton variant="rect" height={200} animation="wave" />
```

Comment:
Let's leave of the `animation="wave"` from every usage. If we want use that animation we can do it once in the theme, but I think the default pulsing is fine for now.